service/controller/resource/monitoring/prometheus/test/case-5-cluster-api-v1alpha3.golden

```diff
@@ -24,13 +24,13 @@ spec:
             values:
             - master
   apiserverConfig:
+    authorization:
+      credentialsFile: /etc/prometheus/secrets/cluster-certificates/token
```

Comment:
Could we also have some test with the certificate auth mechanism?

Reply:
Tests added